libcontainer/container_linux.go

```diff
@@ -919,8 +919,19 @@ func (c *linuxContainer) criuSupportsExtNS(t configs.NamespaceType) bool {
 	return c.checkCriuVersion(minVersion) == nil
 }
 
+// title converts the first character of ASCII string s to uppercase.
+func title(s string) string {
+	s0 := s[0]
```

Comment:
This will panic if string would ever be empty; is that something that could possibly happen? (also could we get non-ascii (multi-byte) characters, in which case we'd be doing the wrong thing?)

Reply:
If the first character is non-ASCI (e.g. хлеб), it will be returned as-is (no capitalization).

Indeed it will panic with empty string as an argument, and it is theoretically possible to have it.

I guess I'll just go back to using strings.Title and suppressing the deprecation warning.

Reply:
Ah, yeah, perhaps best to do that.

I'm considering proposing to un-deprecate this function (I think it's more disruptive than useful)

^^ feel free to beat me to that though if you agree 😁